js/foam/apps/calc/Calc.js

```diff
@@ -376,13 +391,23 @@ CLASS({
       name: 'deg',
       speechLabel: 'switch to degrees',
       translationHint: 'short form for "degrees" calculator mode',
-      code: function() { this.degreesMode = true; }
+      code: function() {
```

Comment:
This code updates the label for the rad/deg label to match the state so chromevox can read the correct label.